lib/Assertions/AssertionLightLogger.h

```diff
@@ -0,0 +1,44 @@
+///////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2022 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Markus Pfeiffer
+////////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include "Basics/CrashHandler.h"
```

Comment:
This include seems to be superfluous.
And according to the comments in https://github.com/arangodb/arangodb/pull/17609/files#diff-a3f3d97412e29d24227a8dd685fb6022bc17036472c9151c603e3895433546feR27-R28, including the CrashHandler here defeats the purpose of the PR.

Reply:
Should rather include
```
#include <exception> // std::terminate
#include <ostream>  // std::flush
#include <sstream>  // std::ostringstream